pages/posts/[...slug].tsx

```diff
@@ -144,10 +149,24 @@ export async function getStaticPaths() {
     process.env.BLOG_POSTS_DYNAMO_TABLE_NAME
   );
 
+  let paths = [];
+  for (let post of postsDynamo) {
+    paths.push({
+      params: {
+        slug: [post.Category, post.PostId],
+      },
+    });
+
+    paths.push({
+      params: {
+        slug: [post.Category],
+      },
+    });
+  }
```

Comment:
I corrected how this is being done. Not sure how it was working before (I think `fallback: true` was keeping the issue concealed) but this method is supposed to return an array of objects that have a `params` key.